centreon/packages/ui/src/Snackbar/index.tsx

```diff
@@ -52,6 +58,15 @@ const Snackbar = React.forwardRef(
       closeSnackbar(id);
     };
 
+    const formatedMessage =
+      typeof message === 'string' ? (
+        <div
+          dangerouslySetInnerHTML={{ __html: DOMPurify.sanitize(message) }}
```

Comment:
Don't use dangerouslySetInnerHTML.
But sanitize your HTML string with this option
`, { USE_PROFILES: { html: true } }` in order to return HTML

Reply:
I changed the used library to sanitize-html